mainnet/README.md

```diff
@@ -45,11 +45,11 @@ Retrieval Setup:
 
 In order for users to retrieve data from your node, you will need to open access to retrieval ports.
 
-Ensure the port specified as `NODE_RETRIEVAL_PORT` in the [.env](./.env.example#L16) has open access to the public internet.
+Ensure the port specified as `NODE_RETRIEVAL_PORT` in the [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/master/mainnet/.env.example#L16) has open access to the public internet.
```

Comment:
This actually doesn't point to `NODE_RETRIEVAL_PORT`. Instead of master, should we link to a fixed branch?

Reply:
Good catch! We can update the line number accordingly.

>Instead of master, should we link to a fixed branch?
Is there a best practice you recommend for this link? The problem arises b/c this README file in eigenda-operator gets retrieved and rendered via [here](https://github.com/Layr-Labs/eigenlayer-docs/blob/main/docs/eigenda/operator-guides/networks/mainnet.mdx) into docs.eigenlayer.xyz.

Reply:
yea latest commit is fine

Reply:
done ✅ 

Changed to specific commit/blob link and fixed the line numbers to match